### PR TITLE
[Merged by Bors] - feat: port Analysis.Complex.Arg

### PR DESCRIPTION
**Pull request by riccardobrasca**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/Analysis/NormedSpace/Ray.lean [hunk 1]
```diff
@@ -43,7 +43,7 @@ theorem norm_sub (h : SameRay ℝ x y) : ‖x - y‖ = |‖x‖ - ‖y‖| := by
   wlog hab : b ≤ a with H
   · rw [SameRay.sameRay_comm] at h
     rw [norm_sub_rev, abs_sub_comm]
-    have := @H E _ _ F
+    have := @H E _ _ ℝ
```
Comment (riccardobrasca) [inline Mathlib/Analysis/NormedSpace/Ray.lean:46]: This introduced a useless `F` in the statement (that is not present in the mathlib3 version).

File: Mathlib/Analysis/Complex/Arg.lean [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+/-!
+# Rays in the complex numbers
+
+This file links the definition `SameRay ℝ x y` with the equality of arguments of complex numbers,
+the usual way this is considered.
+
+## Main statements
+
+* `Complex.sameRay_iff` : Two complex numbers are on the same ray iff one of them is zero, or they
+  have the same argument.
```
Comment (riccardobrasca) [inline Mathlib/Analysis/Complex/Arg.lean:23]: @ericrbg can you have a look at the doc? The corrent mathlib3 doc is inaccurate, and I just removed the wrong line here. Feel free to add a correct one. Thanks!

File: Mathlib/Analysis/NormedSpace/Ray.lean (hunk 1 above)
Comment (ericrbg) [reply Mathlib/Analysis/NormedSpace/Ray.lean:46]: Sorry, I'm not sure how that snuck in!

File: Mathlib/Analysis/Complex/Arg.lean (hunk 2 above)
Comment (ericrbg) [reply Mathlib/Analysis/Complex/Arg.lean:23]: I guess the Lean3 doc says the `Iff.Neg` version, kind of, haha. I'll push a fix in both places.

**Comment by jcommelin** [comment]
bors d+

**Comment by riccardobrasca** [comment]
bors merge

**Pull request closed without merging**